pymc3/progressbar.py

```diff
@@ -1,140 +0,0 @@
-"""
-A simple progress bar to monitor MCMC sampling progress.
-Modified from original code by Corey Goldberg (2010)
-"""
-
-from __future__ import print_function
-
-import warnings
-warnings.simplefilter(action="ignore", category=FutureWarning)
-import sys
-import time
-import datetime
-import uuid
-try:
-    __IPYTHON__
-    from IPython.core.display import HTML, Javascript, display
-except (NameError, ImportError):
-    pass
-
-__all__ = ['progress_bar']
-
-
-class ProgressBar(object):
-
-    def __init__(self, iterations, animation_interval=.5):
-        self.iterations = iterations
-        self.start = time.time()
-        self.last = 0
-        self.animation_interval = animation_interval
-
-    def percentage(self, i):
-        return 100 * i / float(self.iterations)
-
-    def update(self, i):
-        elapsed = time.time() - self.start
-        i += 1
-
-        if elapsed - self.last > self.animation_interval:
-            self.animate(i, elapsed)
-            self.last = elapsed
-        elif i == self.iterations and i != 1:
-            self.animate(i, elapsed)
-
-
-class TextProgressBar(ProgressBar):
-
-    def __init__(self, iterations, printer):
-        self.fill_char = '-'
-        self.width = 20
-        self.printer = printer
-
-        super(TextProgressBar, self).__init__(iterations)
-        self.update(0)
-
-    def animate(self, i, elapsed):
-        self.printer(self.progbar(i, elapsed))
-
-    def progbar(self, i, elapsed):
-        bar = self.bar(self.percentage(i))
-        sps = i / float(elapsed)
-        eta = (self.iterations / sps) - elapsed
-        its = self.iterations
-
-        prog_str = "[{bar}] {it} of {its} in {s_elapsed} sec. " \
```

Comment:
SPS and ETA are kinda nice metrics though. Is there now way to get that with tqdm?